homeassistant/components/sensor/wunderground.py

```diff
@@ -0,0 +1,164 @@
+"""Support for Wunderground weather service."""
+from datetime import timedelta
+import logging
+import requests
+
+import voluptuous as vol
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.config_validation import ensure_list
+from homeassistant.util import Throttle
+from homeassistant.const import (CONF_PLATFORM, CONF_MONITORED_CONDITIONS,
+                                 CONF_API_KEY, TEMP_FAHRENHEIT, TEMP_CELSIUS,
+                                 STATE_UNKNOWN)
+
+CONF_PWS_ID = 'pws_id'
+
+_URL_QUERY = '/conditions/q/'
```

Comment:
Query has a specific meaning when it comes to URI. It is the part that contains the parameters. ie `google.com/?q=hello`. `q=hello` would be the query here.

I also prefer using string formatting. So name it

```
_RESOURCE = 'http://api.wunderground.com/api/{}/conditions/q/'

…

    url = _RESOURCE.format(self._api_key)
```